package.json

```diff
@@ -9,9 +9,9 @@
   },
   "scripts": {
     "start": "node main.js",
-    "test": "mocha -w",
+    "test": "mocha --recursive -w",
```

Comment:
This needs attention - because `PlayerActions` tests are in a folder, Mocha can't find them unless we recurse or provide a directory.. It looks like this is throwing some error, but it doesn't say what. Not sure if it has to do with the `coverage` command's employment of it, though, as I was really using that one more than this one.